File: Assignment 1/avgoffive.cpp
#include<iostream>
using namespace std;
class calculator
{
    int num1,num2,num3,num4,num5,result;
    public:void avg()
    {
        cout<<"enter first no:\n";
        cin>>num1;
        cout<<"enter second no:\n";
        cin>>num2;
        cout<<"enter third no:\n";
        cin>>num3;
        cout<<"enter four no:\n";
        cin>>num4;
        cout<<"enter fifth no:\n";
        cin>>num5;
        result=(num1+num2+num3+num4+num5)/5;
        cout<<"result:\n"<<result;
    }
};
int main()
{
    calculator value;
    value.avg();
}

File: JAVA/Assignment2/AvgOfFive.java
import java.util.Scanner ;
public class AvgOfFive
{
   public static void main(String args[])
   {
	  float num1,num2,num3,num4,num5;
	  float avg=0;
      System.out.println("Enter 5 no's::");
      Scanner sc = new Scanner(System.in);
	  num1=sc.nextInt();
	  num2=sc.nextInt();
	  num3=sc.nextInt();
	  num4=sc.nextInt();
	  num5=sc.nextInt();
	  avg=(num1+num2+num3+num4+num5)/5;
		 System.out.println("The avg value is ="+avg); 
   }
}